presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -652,10 +653,10 @@ public void testIn()
         assertQuery("SELECT orderkey FROM orders WHERE totalprice IN (1, 2, 3)");
     }
 
-    @Test
-    public void testLargeIn()
+    @Test(dataProvider = "largeInValuesCount")
+    public void testLargeIn(int valuesCount)
     {
-        String longValues = range(0, 5000)
+        String longValues = range(0, valuesCount)
                 .mapToObj(Integer::toString)
                 .collect(joining(", "));
         assertQuery("SELECT orderkey FROM orders WHERE orderkey IN (" + longValues + ")");
```

Comment:
would it be possible to calc expected values without h2? would that improve test runtime?

like:
```
assertThat(query("....."))
  .matches( materialized result )
```



(treat this optional)

Reply:
I will skip this one. Materialized rows for "not in" are kinda painful.